soc/nordic/nrf53/Kconfig

```diff
@@ -215,19 +219,49 @@ config SOC_ENABLE_LFXO
 choice SOC_LFXO_LOAD_CAPACITANCE
 	prompt "LFXO load capacitance"
 	depends on SOC_ENABLE_LFXO
-	default SOC_LFXO_CAP_INT_7PF
```

Comment:
Why is this removed? Behavior of these deprecated options should not be changed.

Reply:
likely a mistake, re-added